simulator.py

```diff
@@ -0,0 +1,62 @@
+# Class - Simulator
+# Runs multiple simulations of the game and saves the results
+
+import game
+import multiprocessing as mp
+import csv
+
+class Simulator:
+    """Class that manages simulations of a BlackJack game
+    
+    Parameters:
+
+    sim_count (int): Indicated the number of simulations to compute
+
+    """
+
+    def __init__(self, sim_count=100, bet_amount=1):
```

Comment:
What are reasonable defaults for these values?

Reply:
Bet amount looks good.

I think we still need to play around with the sim_count but for now 10 thousand should be the minimum I think.

Reply:
Running 100k tests vary between a percent in the final result, and they last between 25-30 seconds. I think it would be a good default value. With 1 million tests the EV becomes closer to what we're looking for, though it takes roughly 4 minutes. (these times are without concurrency)